recipe/meta.yaml

```diff
@@ -59,12 +59,14 @@ outputs:
         - natsort
         - nc-time-axis
         - netCDF4
+        - numba
         - numpy !=1.24.3  # severe masking bug
         - openpyxl
         - packaging
         - pandas
         - progressbar2
         - prov
+        - psyplot
         - psy-maps
         - psy-reg
         - psy-simple
```

Comment:
In environment.yml we have `shapely >=2` (I cannot comment on that line directly).

Reply:
Not yet, adding it in https://github.com/ESMValGroup/ESMValTool/pull/3487. But we anyway have the pin via ESMValCore

Reply:
True, I compared against https://github.com/ESMValGroup/ESMValTool/pull/3487.